addons/pinniped/post-deploy/pkg/vars/vars.go

```diff
@@ -45,6 +45,8 @@ var (
 	// JWTAuthenticatorName is the JWT authentication name.
 	JWTAuthenticatorName = ""
 
+	JWTAuthenticatorAudience = ""
```

Comment:
comment to match over vars?

Reply:
I'll add.
As an aside, these are kind of terrible comments (just restating the var name) 😄

Reply:
I'll add.
As an aside, these are kind of terrible comments (just restating the var name) 😄